db/system_keyspace.cc

```diff
@@ -256,11 +256,13 @@ schema_ptr system_keyspace::topology() {
             .with_column("num_tokens", int32_type)
             .with_column("shard_count", int32_type)
             .with_column("ignore_msb", int32_type)
+            .with_column("supported_features", set_type_impl::get_instance(utf8_type, true))
             .with_column("new_cdc_generation_data_uuid", uuid_type, column_kind::static_column)
             .with_column("transition_state", utf8_type, column_kind::static_column)
             .with_column("current_cdc_generation_uuid", uuid_type, column_kind::static_column)
             .with_column("current_cdc_generation_timestamp", timestamp_type, column_kind::static_column)
             .with_column("global_topology_request", utf8_type, column_kind::static_column)
+            .with_column("enabled_features", set_type_impl::get_instance(utf8_type, true))
```

Comment:
Should be static, no?

Reply:
Yes, definitely, I'll change it.

Reply:
I mentioned it in the cover letter:

> `enabled_features set<text>` is a new static column which holds the features that are considered enabled by the cluster. Unlike in the current gossip-based implementation the features will not be enabled implicitly when all nodes support a feature, but rather via an explicit action of the topology coordinator.

Reply:
Ok. Sorry for not noticing (I'm really good at asking to put stuff in cover letters, but very bad at actually reading them). It makes sense for the enabling to be an explicit step (since some transactional action might be needed, like changing the schema).